src/components/finder/utils.ts

```diff
@@ -21,7 +21,9 @@ export const makeColumns = (doc: LocalDocumentData, pathname: string | null) =>
 
     if (doc.parent) {
         let node = doc
-        while (node.parent) {
+        // we have to restrict the length here as well due to docAPI caching/memoization
+        // which will return all parent hierarchy fetched so far 
```

Comment:
that was intended to show all the hierarchy up to the root directory there

Reply:
I understand, but that is the scope of this task: https://github.com/CRJI/EIC/issues/995. We want to limit the hierarchy to a maximum of 4-5 columns. Let me know if I was wrong :)